extensions/servlet/src/com/google/inject/servlet/ServletScopes.java

```diff
@@ -218,9 +220,15 @@ public String toString() {
    *     thread, or if the request has completed.
    * 
    * @since 3.0
+   * @deprecated You probably want to use {@code transferRequest} instead
```

Comment:
We've found continueRequest to have a variety of issues.  Internally, some teams have taken to using a rewritten `@RequestScope` implementation that allows access through background threads (for types that are guaranteed to be immutable, or whitelisted types).

If continueRequest is working for you, no reason to stop using it... but be aware that it makes it easy to accidentally access mutable objects from many threads at once.